UI/ProductionWnd.cpp

```diff
@@ -467,10 +467,17 @@ namespace {
         // get location indicator text
         std::string location_text;
         bool system_selected = false;
+        bool rally_dest_selected = (elem.rally_point_id != -1 && elem.rally_point_id == SidePanel::SystemID());
         if (std::shared_ptr<const UniverseObject> location = GetUniverseObject(elem.location)) {
             system_selected = (location->SystemID() != -1 && location ->SystemID() == SidePanel::SystemID());
-            if (GetOptionsDB().Get<bool>("ui.queue.production_location.shown"))
-                location_text = boost::io::str(FlexibleFormat(UserString("PRODUCTION_QUEUE_ITEM_LOCATION")) % location->Name()) + " ";
+            if (GetOptionsDB().Get<bool>("ui.queue.production_location.shown")) {
+                if (rally_dest_selected && !system_selected) {
+                    location_text = boost::io::str(FlexibleFormat(UserString(
+                                    "PRODUCTION_QUEUE_ITEM_RALLIED_FROM_LOCATION")) % location->Name()) + " ";
```

Comment:
indent additional 4 spaces or to match the previous line's parameters (which there are none of in this case)

Reply:
So, lining up under the "t" in location, correct?

Reply:
Yep